test/parallel/test-zlib-random-byte-pipes.js

```diff
@@ -154,23 +150,9 @@ const gunz = zlib.createGunzip();
 
 inp.pipe(gzip).pipe(gunz).pipe(out);
 
-inp.on('data', function(c) {
-  console.error('inp data', c.length);
-});
-
-gzip.on('data', function(c) {
-  console.error('gzip data', c.length);
-});
-
-gunz.on('data', function(c) {
-  console.error('gunz data', c.length);
-});
-
-out.on('data', function(c) {
-  console.error('out data', c.length);
-});
-
-out.on('data', common.mustCall(function(c) {
-  console.error('hash=%s', c);
+inp.on('data', common.mustCall(4));
```

Comment:
I know nothing of this test, but empirical testing shows that there is no guarantee that this will be called 4 times. Under load at least, it may be called 5 times. 

Given that this is all from ` inp.pipe(gzip).pipe(gunz).pipe(out)` and we have a check later for the `data` event on `out`, I think this can just be removed. Probably the same for the `gunz` and `gzip` checks below.